charts/openshift-metering/templates/reporting-operator/reporting-operator-deployment.yaml

```diff
@@ -441,6 +446,13 @@ spec:
 {{- end }}
 {{- end }}{{/* end of authProxy.enabled */}}
       volumes:
+{{- if and .Values.networking.useGlobalProxyNetworking .Values.networking.proxy.config.trusted_ca_bundle }}
+      - name: reporting-operator-trusted-ca-bundle
+        valueFrom:
+          configMapKeyRef:
+            name: reporting-operator-trusted-ca-bundle
```

Comment:
Let's add a key to the `reporting-operator-config` ConfigMap that points to the `/etc/pki/ca-trust/extracted/pem/ca-bundle.crt`, and change the `name: reporting-operator-trusted-ca-bundle` to `name: reporting-operator-config`, and the `key` value is now the name of the variable that points to the `ca-bundle.crt`.